hack/install-tools.sh

```diff
@@ -1,11 +1,13 @@
 #!/bin/bash
 
-set -e
+set -o errexit
+set -o nounset
+set -o pipefail
 
-STI_ROOT=$(dirname "${BASH_SOURCE}")/..
-source "${STI_ROOT}/hack/common.sh"
+STARTTIME=$(date +%s)
 
-GO_VERSION=($(go version))
-echo "Detected go version: $(go version)"
+echo $(go version)
 
-go get golang.org/x/tools/cmd/cover github.com/tools/godep
+go get github.com/tools/godep github.com/golang/lint/golint
+
+ret=$?; ENDTIME=$(date +%s); echo "$0 took $(($ENDTIME - $STARTTIME)) seconds"; exit "$ret"
```

Comment:
Just copying what is in origin :yum: 

If we will talk about it, I'd point out another problem is that when ret != 0 we never see that line printed because of `set -e`....

I didn't intend this PR to go fix that (this I would like to apply to all scripts, here and in origin). Feel free to fix it with more bash-foo.